Documentation/install/azure/azure-terraform.md

```diff
@@ -178,16 +178,30 @@ $ mkdir -p build/${CLUSTER}
 $ cp examples/terraform.tfvars.azure build/${CLUSTER}/terraform.tfvars
 ```
 
-Edit the parameters in `build/$CLUSTER/terraform.tfvars` with the deployment's Azure details, domain name, license, and pull secret. [View all of the Azure specific options][azure-vars] and [the common Tectonic variables][vars].
-
 ### Key values for basic Azure deployment
 
-These are the basic values that must be adjusted for each Tectonic deployment on Azure. See the details of each value in the [terraform.tfvars][terraform-tvars] file.
+These are the basic values that must be adjusted for each Tectonic deployment on Azure.
+
+#### Environment variables
+
+Set these sensitive values in the shell environment, from where Terraform will encrypt them before storing them in cluster state files:
+
+* `TF_VAR_tectonic_admin_email` - String giving the email address used as user name for the initial Console login
```

Comment:
Random drive by opinion on this layout. If we did something like this, we get values that are copy-pasteable, with default values, and the description:

> Set the email address used as user name for the initial Console login: 
> ```
> export TF_VAR_tectonic_admin_email=admin@example.com
> ```
>
> Enter the plaintext password string for initial Console login. This will be encrypted via Terraform:
> ```
> export TF_VAR_tectonic_admin_password_hash=abc123
> ```
>
> ...

Reply:
You could argue that it makes more sense to maybe group them together, as well:

> Set the email address used as user name and plaintext password string for initial Console login. This will be encrypted via Terraform:
> ```
> export TF_VAR_tectonic_admin_email=admin@example.com
> export TF_VAR_tectonic_admin_password_hash=abc123
> ```

Reply:
Agreed with @robszumski; we should make these copy-pasteable.
It might be nice to have them all as separate blocks, so people don't blindly c/p default values for sensitive creds.

In addition, all of these should be encased with double quotes:
> `export TF_VAR_tectonic_admin_email="admin@example.com"`

vs

> `export TF_VAR_tectonic_admin_email=admin@example.com`